src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,32 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Line {
+
+  private final List<Point> points = new ArrayList<>();
+
+  public Line(int countOfPerson) {
+    IntStream.range(0, countOfPerson)
+        .forEach(i -> points.add(connectPoint(new RandomConnect())));
```

Comment:
```suggestion
  private final List<Point> points;

  public Line(int countOfPerson) {
    points = IntStream.range(0, countOfPerson)
        .map(...)
        .collect(...);
```

여기도 굳이 수정해주실 필요는 없지만, IntStream 의 forEach 보다는 map 을 활용하는게 제 취향이라서 언급해봅니다.
제 취향이므로 꼭 수정해주실 필요는 없습니다.
제 취향에 동의하지 않는 분들도 있어서요.

참고로 아래에서, `points.get(points.size() - 1);` 를 사용하고 있으므로, ArrayList 를 명시해주는 편이 훨씬 효율적일 수도 있겠네요.
그냥 이런 취향인 사람도 있구나 하고 넘어가주시면 됩니다.